src/open_inwoner/accounts/views/contactmoments.py

```diff
@@ -197,14 +198,28 @@ def get_context_data(self, **kwargs):
             raise ImproperlyConfigured("Unknown KlantenServiceType")
 
         origin = self.request.headers.get("Referer")
-        question, zaak_with_api_group = service.retrieve_question(
+        question, zaken_with_api_group = service.retrieve_question(
             self.get_fetch_params(service),
             question_uuid=kwargs["kcm_uuid"],
             user=self.request.user,
         )
         if not question:
             raise Http404()
+        if zaken_with_api_group and len(zaken_with_api_group) > 1:
+            # In principle this should not happen, a zaak should be stored in
+            # exactly one backend. But: https://www.xkcd.com/2200/
+            # We should at least receive a ping if this happens and warn the user.
+            logger.error("Found one zaak in multiple backends")
+            messages.warning(
+                self.request,
+                _(
+                    "The question was found to be related to several zaken, but we cannot "
+                    "determine which zaak is the most relevant"
+                ),
+            )
```

Comment:
Perhaps:

```suggestion
            messages.warning(
                self.request,
                _(
                    "Your question was connected to multiple cases, we are showing you the first."
                ),
            )
```

I think the point of the message is to let the user know the page might not meet their prior expectations, rather than letting them know what _our_ problem is.